.mailmap

```diff
@@ -781,6 +781,8 @@ Jason Siefken <siefkenj@gmail.com>
 Jason Tokayer <jason.tokayer@gmail.com>
 Jason Tokayer <jason.tokayer@gmail.com> <jason.tokayer@capitalone.com>
 Jatin Bhardwaj <bhardwajjatin093@gmail.com> <148186488+Jatinbhardwaj-093@users.noreply.github.com>
+Jatin Gaur <jatin22101@iiitnr.edu.in> <jatin22101@iiitnr.edu.in>
+Jatin Gaur <jatingaur18@users.noreply.github.com> <134034110+jatingaur18@users.noreply.github.com>
```

Comment:
Change this to something like:
```
Jatin Gaur <jatin22101@iiitnr.edu.in> <jatin22101@iiitnr.edu.in>
Jatin Gaur <jatin22101@iiitnr.edu.in> <134034110+jatingaur18@users.noreply.github.com>
```
Both lines should begin with the same name and email address and that will be the name that is recorded in the AUTHORS file:
https://github.com/sympy/sympy/blob/e17302a7e58f564550f0517061bf9ad376e5957f/.mailmap#L138-L148